pymodbus/server/async_io.py

```diff
@@ -166,9 +166,10 @@ async def server_async_execute(self, request, *addr):
                 response = await request.update_datastore(context)
 
         except NoSuchSlaveException:
-            Log.error("requested slave does not exist: {}", request.slave_id)
```

Comment:
You remove one line and add the same information in 2 lines, that does not make sense ? 

the app knows if it has requested slaves to be ignored.


Secondly this change seems unrelated to the title of this PR.